lua/apisix/plugins/proxy-rewrite.lua

```diff
@@ -0,0 +1,69 @@
+local core        = require("apisix.core")
+local plugin_name = "proxy-rewrite"
+local pairs       = pairs
+local ipairs      = ipairs
+
+local schema = {
+    type = "object",
+    properties = {
```

Comment:
http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.5.2

Make sure the user has at least one of them enabled.